examples/video-chat/tsconfig.json

```diff
@@ -3,6 +3,7 @@
   "compilerOptions": {
     "paths": {
       "@fishjam-dev/react-native-client": ["../src/"]
-    }
+    },
+    "types": ["node"]
```

Comment:
is this necessary?

Reply:
Yes, it fixes issue with `process.env` typings